Sumo Orgs Flex docs (#4962)
SumoLogic/sumologic-documentation: docs/manage/manage-subscription/create-manage-orgs-flex.md

---
id: create-manage-orgs-flex
title: Create and Manage Orgs (Flex)
description: Learn how to create and manage multiple Sumo Logic orgs (Flex).
---

import useBaseUrl from '@docusaurus/useBaseUrl';

## Availability

| Account Type | Account Level |
|:--|:--|
| Credits | Essentials, Enterprise Suite Flex |

:::note

> This feature is not enabled by default. If you’d like to have it enabled, contact your Sumo Logic Account Executive.

Reviewer: ```suggestion
This feature is not enabled by default. If you’d like to have it enabled, contact your Sumo Logic account executive.
```

:::

> This topic has information about Sumo Logic’s Organizations (“Sumo Orgs”) feature for Flex, which you can use to create and manage orgs. We use the term *parent org* to refer to the org from which you create a new org, and *child orgs* to refer to the orgs you create. 

Reviewer: ```suggestion
This topic has information about Sumo Logic’s Organizations (“Sumo Orgs”) feature for Flex, which you can use to create and manage multiple orgs. The term *parent org* refers to the organization from which you create a new organization, while *child orgs* are the organizations you create.
```

Users that have the required role capabilities (described in the following section) can create child orgs under a parent org, and manage and monitor the allocation and consumption of Sumo Logic credits across orgs, and for each child org. This functionality is available in the Sumo Logic UI in the **Organizations** tab and also in the [Organizations Management API](https://organizations.sumologic.com/docs/).

## About Cloud SIEM provisioning

You can enable and provision Cloud SIEM for a child org. Note that the process of provisioning Cloud SIEM can take up to 24 hours. For more information, see [Monitor Cloud SIEM Provisioning](#monitor-cloud-siem-provisioning), below.

## Requirements for creating and managing orgs

> There are several role capabilities that are required to work with orgs:

Reviewer: ```suggestion
There are several [role capabilities](/docs/manage/users-roles/roles/role-capabilities) that are required to work with orgs:
```

* **View Organizations**. This capability is required to view the Organizations UI.
* **Create Organizations**. This capability is required to create or provision child organizations.
* **Change Credits Allocation**. This capability is required to change the credits allocation and baselines for a child organization. 
* **Create Trial Organizations**. This capability is required to create and provision trial organizations. 
* **Upgrade Trial Organizations**. This capability is required to upgrade trial organizations. 

## Create a new child organization

This section has instructions for creating a new org.

:::note

> After you create a new org, you can’t delete it.

Reviewer: should this say _new child org_?
